app/command.go

```diff
@@ -519,6 +519,17 @@ func (a *App) DoCommandRequest(cmd *model.Command, p url.Values) (*model.Command
 }
 
 func (a *App) HandleCommandResponse(command *model.Command, args *model.CommandArgs, response *model.CommandResponse, builtIn bool) (*model.CommandResponse, *model.AppError) {
+	if err := response.IsValid(); !builtIn && err != nil {
+		mlog.Debug("Invalid command response", mlog.String("errors", err.Error()))
+		return response, model.NewAppError(
+			"HandleCommandResponse",
+			"Invalid command response",
```

Comment:
Can you add an error string in the `en.json` file and use it here(similar to line 549).

Reply:
Yep!